### ISSUE TITLE
[CRASH] "Well This is Awkward" screen on editing channel emoji

### ISSUE THREAD

**Issue by Riddim-GLiTCH** [arg0NNY/DiscordPlugins#108]
### Plugin:

📜 BetterChannelList

### What version of BetterDiscord are you using?

Discord Core: canary 330863 (0b0bea6) - Host 1.0.453 x64 (53084) | BetterDiscord Core: Gtihub Release v1.10.2

### What version of the selected plugin(s) are you using (if applicable)?

v1.1.5

### What client are you using?

🟠 Canary

### Describe the bug

BetterChannelsList crashes the discord client, resulting in the "Well This is Awkward" Error screen, When clicking on the option to change the channel emoji.

Steps to Reproduce:
1. Start Discord with BetterDiscord injected, And the plugin in question installed and enabled.
2. Navigate to any server.
3. Right click a channel and select the context menu item "Edit Emoji Icon".

Expected Behavior: 
- The Discord Client will panic, and a crash handler screen is shown.
- The log outputs the following errors:
<details>
<summary>Log Output</summary>

```log
web.68f269ad1f095020bf9e.js:12 [BetterDiscord] [Patcher] Could not fire instead callback of Z for BetterChannelList TypeError: Cannot destructure property 'channel' of 'props' as it is undefined.
    at Object.eval [as callback] (BetterChannelList.plugin.js:938:21)
    at renderer.js:5:22680
    at q (web.68f269ad1f095020bf9e.js:12:1265828)
    at Q (web.68f269ad1f095020bf9e.js:12:1266487)
    at og (web.68f269ad1f095020bf9e.js:12:1331051)
    at lr (web.68f269ad1f095020bf9e.js:12:1369815)
    at lo (web.68f269ad1f095020bf9e.js:12:1370583)
    at s8 (web.68f269ad1f095020bf9e.js:12:1367684)
    at iT (web.68f269ad1f095020bf9e.js:12:1306525)
    at l_ (web.68f269ad1f095020bf9e.js:12:1372300)
    at ld (web.68f269ad1f095020bf9e.js:12:1371265)
    at s3 (web.68f269ad1f095020bf9e.js:12:1366210)
    at v (web.68f269ad1f095020bf9e.js:12:1555532)
    at L (web.68f269ad1f095020bf9e.js:12:1556062)
    at T.C.<computed> (web.68f269ad1f095020bf9e.js:1:1271621)
    at b (web.68f269ad1f095020bf9e.js:1:1271386)
    at MessagePort.D (web.68f269ad1f095020bf9e.js:1:1271445)
S.t.<computed> @ web.68f269ad1f095020bf9e.js:12
(anonymous) @ sentry.24c4099d62150c39f62a.js:14
_log @ renderer.js:5
err @ renderer.js:5
(anonymous) @ renderer.js:5
q @ web.68f269ad1f095020bf9e.js:12
Q @ web.68f269ad1f095020bf9e.js:12
og @ web.68f269ad1f095020bf9e.js:12
lr @ web.68f269ad1f095020bf9e.js:12
lo @ web.68f269ad1f095020bf9e.js:12
s8 @ web.68f269ad1f095020bf9e.js:12
iT @ web.68f269ad1f095020bf9e.js:12
l_ @ web.68f269ad1f095020bf9e.js:12
ld @ web.68f269ad1f095020bf9e.js:12
s3 @ web.68f269ad1f095020bf9e.js:12
v @ web.68f269ad1f095020bf9e.js:12
L @ web.68f269ad1f095020bf9e.js:12
T.C.<computed> @ web.68f269ad1f095020bf9e.js:1
b @ web.68f269ad1f095020bf9e.js:1
D @ web.68f269ad1f095020bf9e.js:1
web.68f269ad1f095020bf9e.js:12 [BetterDiscord] [Patcher] Could not fire instead callback of Z for BetterChannelList TypeError: Cannot destructure property 'channel' of 'props' as it is undefined.
    at Object.eval [as callback] (BetterChannelList.plugin.js:938:21)
    at renderer.js:5:22680
    at q (web.68f269ad1f095020bf9e.js:12:1265828)
    at Q (web.68f269ad1f095020bf9e.js:12:1266487)
    at og (web.68f269ad1f095020bf9e.js:12:1331051)
    at lr (web.68f269ad1f095020bf9e.js:12:1369815)
    at lo (web.68f269ad1f095020bf9e.js:12:1370583)
    at s4 (web.68f269ad1f095020bf9e.js:12:1366933)
    at s8 (web.68f269ad1f095020bf9e.js:12:1367739)
    at iT (web.68f269ad1f095020bf9e.js:12:1306525)
    at l_ (web.68f269ad1f095020bf9e.js:12:1372300)
    at ld (web.68f269ad1f095020bf9e.js:12:1371265)
    at s3 (web.68f269ad1f095020bf9e.js:12:1366210)
    at v (web.68f269ad1f095020bf9e.js:12:1555532)
    at L (web.68f269ad1f095020bf9e.js:12:1556062)
    at T.C.<computed> (web.68f269ad1f095020bf9e.js:1:1271621)
    at b (web.68f269ad1f095020bf9e.js:1:1271386)
    at MessagePort.D (web.68f269ad1f095020bf9e.js:1:1271445)
S.t.<computed> @ web.68f269ad1f095020bf9e.js:12
(anonymous) @ sentry.24c4099d62150c39f62a.js:14
_log @ renderer.js:5
err @ renderer.js:5
(anonymous) @ renderer.js:5
q @ web.68f269ad1f095020bf9e.js:12
Q @ web.68f269ad1f095020bf9e.js:12
og @ web.68f269ad1f095020bf9e.js:12
lr @ web.68f269ad1f095020bf9e.js:12
lo @ web.68f269ad1f095020bf9e.js:12
s4 @ web.68f269ad1f095020bf9e.js:12
s8 @ web.68f269ad1f095020bf9e.js:12
iT @ web.68f269ad1f095020bf9e.js:12
l_ @ web.68f269ad1f095020bf9e.js:12
ld @ web.68f269ad1f095020bf9e.js:12
s3 @ web.68f269ad1f095020bf9e.js:12
v @ web.68f269ad1f095020bf9e.js:12
L @ web.68f269ad1f095020bf9e.js:12
T.C.<computed> @ web.68f269ad1f095020bf9e.js:1
b @ web.68f269ad1f095020bf9e.js:1
D @ web.68f269ad1f095020bf9e.js:1
web.68f269ad1f095020bf9e.js:12 Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at lR (web.68f269ad1f095020bf9e.js:12:1376506)
    at _ (web.68f269ad1f095020bf9e.js:12:1317558)
    at h (web.68f269ad1f095020bf9e.js:12:1318913)
    at m (web.68f269ad1f095020bf9e.js:12:1320965)
    at oD (web.68f269ad1f095020bf9e.js:12:1332627)
    at l (web.68f269ad1f095020bf9e.js:12:1381021)
    at lu (web.68f269ad1f095020bf9e.js:12:1370772)
    at ls (web.68f269ad1f095020bf9e.js:12:1370700)
    at lo (web.68f269ad1f095020bf9e.js:12:1370563)
    at s4 (web.68f269ad1f095020bf9e.js:12:1366933)
    at s8 (web.68f269ad1f095020bf9e.js:12:1367739)
    at iT (web.68f269ad1f095020bf9e.js:12:1306525)
    at l_ (web.68f269ad1f095020bf9e.js:12:1372300)
    at ld (web.68f269ad1f095020bf9e.js:12:1371265)
    at s3 (web.68f269ad1f095020bf9e.js:12:1366210)
    at v (web.68f269ad1f095020bf9e.js:12:1555532)
    at L (web.68f269ad1f095020bf9e.js:12:1556062)
    at T.C.<computed> (web.68f269ad1f095020bf9e.js:1:1271621)
    at b (web.68f269ad1f095020bf9e.js:1:1271386)
    at MessagePort.D (web.68f269ad1f095020bf9e.js:1:1271445)
```

</details>

### Additional comments

This issue was first discussed in [this betterdiscord support thread](https://discord.com/channels/86004744966914048/1288942544922607669). Please note that OP also mentioned another issue with an unrelated plugin. Which is also discussed in that thread.

### I have read and agree to the following:

- [X] I have searched existing/closed issues to confirm that this hasn't been solved already.
- [X] I've already tried to disable all other plugins (or themes) to confirm this issue is related to the plugin.
This isn't necessarily required

**Comment by TheCraftedGamer** [comment]
this applies to base discord as well, on start up or loading of the plugin

**Comment by arg0NNY** [comment]
Fixed in cedca17bdddc43f7a20343855758042124f0dd78
